feat: fuse-overlayfs configuration (#2685)
codenvy/che-docs: modules/end-user-guide/pages/using-the-fuse-overlay-storage-driver.adoc

:_content-type: CONCEPT
:description: Using the fuse-overlayfs storage driver for Podman
:keywords: authentication, authenticate, github, gitlab, bitbucket
:navtitle: Using the fuse-overlayfs storage driver for Podman
:page-aliases:

[id="using-the-fuse-overlay-storage-driver-for-podman"]
= Using the fuse-overlayfs storage driver for Podman

By default, newly created workspaces that do not specify a devfile will use the Universal Developer Image (UDI).
The UDI contains common development tools and dependencies commonly used by developers.

Podman is included in the UDI, allowing developers to build and push container images from their workspace.

By default, Podman in the UDI is configured to use the `vfs` storage driver.

> For more efficient image management, it is recommended to use the fuse-overlayfs storage driver which supports copy-on-write in rootless environments.

Reviewer: ```suggestion
For more efficient image management, use the fuse-overlayfs storage driver which supports copy-on-write in rootless environments.
```

> To use fuse-overlayfs in a workspace, there are three requirements:

Reviewer: ```suggestion
You must meet the following requirements to fuse-overlayfs in a workspace:
```
